src/scss/table.scss

```diff
@@ -0,0 +1,14 @@
+.table-headers.table-headers {
+  background-color: #26a69a;
+  color: white;
+}
+
+.first-item.first-item {
```

Comment:
Why do you have duplicate classes here and upper(table-headers)?

Reply:
it some feature to get more priority for selector to override library's styles.